crates/ruff/src/rules/flake8_pyi/rules/simple_defaults.rs

```diff
@@ -28,6 +28,14 @@ impl Violation for ArgumentSimpleDefaults {
     }
 }
 
+const ALLOWED_MATH_ATTRIBUTES_IN_DEFAULTS: &[&[&str]] = &[
+    &["math", "inf"],
+    &["math", "nan"],
+    &["math", "e"],
+    &["math", "pi"],
+    &["math", "tau"],
+];
+
 const ALLOWED_ATTRIBUTES_IN_DEFAULTS: &[&[&str]] = &[
```

Comment:
Nit: let's rename this to `ALLOWED_SYS_ATTRIBUTES_IN_DEFAULTS`?

Reply:
This follows the variable name in flake8-pyi. I think nothing need to change here. This list may extend in the future (e.g., `os.path.sep`). A separate variable for `math.*` due to they support `-` op (`-math.inf`).

Reply:
Ah ok, that's reasonable.